bin/rename-rc-vars

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env bash
+# shellcheck source-path=..
+
+set -euo pipefail
+
+#### Detect Toolkit Project Root ####
+# if realpath is not available, create a semi-equivalent function
+command -v realpath >/dev/null 2>&1 || realpath() {
+  [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"
+}
+SCRIPT_PATH="$(realpath "${BASH_SOURCE[0]}")"
+SCRIPT_DIR="$(dirname "$SCRIPT_PATH")"
+TOOLKIT_ROOT="$(realpath "$SCRIPT_DIR/..")"
+if [[ ! -d "$TOOLKIT_ROOT/bin" ]] || [[ ! -d "$TOOLKIT_ROOT/config" ]]; then
+  echo "ERROR: could not find root of overleaf-toolkit project (inferred project root as '$TOOLKIT_ROOT')"
+  exit 1
+fi
+
+source "$TOOLKIT_ROOT/lib/shared-functions.sh"
+
+function usage() {
+  echo "Usage: bin/rename-rc-vars"
```

Comment:
```suggestion
  echo "Usage: bin/rename-rc-vars-5-0"
```
Similar to the variables script, it might be worth adding a version suffix here, just in case we need to migrate entries in the future.

Reply:
The `overleaf.rc` is not linked to the version of Server Pro (we don't check the version in `bin/upgrade` before updating it), that's why I didn't add the suffix.

Reply:
Good point!